state_version_output_integration_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const waitForStateVersionOutputs = 700 * time.Millisecond
+const waitForStateVersionOutputs = 1000 * time.Millisecond
```

Comment:
why 1000 instead of 700?

Reply:
My tests didn't pass locally with the old timing.